### ISSUE TITLE
Choose the frontend technology

### ISSUE THREAD

**Issue by krzysztofrewak** [blumilksoftware/internships#4]
In my opinion the entire application should be a single page application, but in the easiest way: like only one page with some navigation and sections and all information there. Nontheless we should decide how are we going to proceed with this dynamic list of companies and the interactive map.

I would like to hear (or read) some cons and pros of [vanilla JavaScript](http://vanilla-js.com/), simple things like [Handlebars](https://handlebarsjs.com/) and more complex frameworks like [Vue](https://vuejs.org/), [React](https://reactjs.org/), [Angular](https://angular.io/) or [Svelte](https://svelte.dev/).

The result of this task should be to choose one technology.

**Comment by LukaszPodlipski** [comment]
I believe Vue JS would be smart choice. We all have to learn from scratch this framework, so its smth new and challenging. Also Vue is fast and has low entry level.

**Comment by brzozka** [comment]
After investigating our options I think Vue will be the best option to choose in our project. It will be great opportunity for us to learn and use one of the JS frameworks in this app to gain some experience, that's why vanilla JS won't be something we consider working with. Handlebars is pure rendering engine so in this case we would have to use it in addition with something, and I think Vue provides us with similar funcionality when it comes to templates. 

Regarding more complex frameworks I've read that Vue is really begginer-friendly with simple syntax and learning curve. It's not as popular as React, but it's more suited for single page web-apps and smaller projects. That's also the case with Angular, It might be great for big projects/apps but can get really overwhelming for begginers and smaller teams. Svelte seems interesting but it might not be the best choice as a first frontend technology to learn. It's rather new, and not that popular at the moment, so it can be difficult to find documentation/materials for things we need. 

In conclusion I think Vue is the best choice for our app.
